api/src/main/java/org/openmrs/module/openhmis/cashier/web/CashierWebConstants.java

```diff
@@ -35,4 +35,6 @@
 	public static final String OPENHMIS_CASHIER_MODULE_ID = "openhmis.cashier";
 	public static final String LANDING_PAGE_EXTENSION_POINT_ID = "org.openmrs.module.openhmis.cashier.landing";
 	public static final String MANAGE_MODULE_PAGE_EXTENSION_POINT_ID = "org.openmrs.module.openhmis.cashier.manage.module";
+	public static final String MESSAGE_PROPERTIES_JS_URI = MODULE_ROOT + "cashierMessageProperties.js";
```

Comment:
Why doesn't this follow the <NAME>_ROOT and <NAME>_PAGE pattern established above?